Makefile.inc1

```diff
@@ -537,12 +537,12 @@ SRCRELDATE!=	awk '/^\#define[[:space:]]*__FreeBSD_version/ { print $$3 }' \
 VERSION=	FreeBSD ${_REVISION}-${_BRANCH:C/-p[0-9]+$//} ${TARGET_ARCH} ${SRCRELDATE}
 .export VERSION
 .endif
+MAJOR_REVISION=	${_REVISION:R}
 
 .if !defined(PKG_VERSION)
 _STRTIMENOW=	%Y%m%d%H%M%S
 _TIMENOW=	${_STRTIMENOW:gmtime}
 .if ${_BRANCH:MCURRENT*} || ${_BRANCH:MSTABLE*} || ${_BRANCH:MPRERELEASE*}
-_REVISION:=	${_REVISION:R}
```

Comment:
This changes what xdev links happen, and it's my belief they are now more correct since they are supposed to have major.minor in them....

There's only one other use of _REVSION other than below I could find:
```
package-pkg: .PHONY
        rm -rf /tmp/ports.${TARGET} || :
        env ${WMAKEENV:Q} SRCDIR=${.CURDIR} PORTSDIR=${PORTSDIR} REVISION=${_REVISION} \
                PKG_CMD=${PKG_CMD} PKG_VERSION=${PKG_VERSION} REPODIR=${REPODIR} \
                WSTAGEDIR=${WSTAGEDIR} \
                sh ${.CURDIR}/release/scripts/make-pkg-package.sh

```
and make-pkg-package.sh just uses it for
```
        CONFIGURE_ARGS="--host=$(uname -m)-portbld-freebsd${REVISION}" \
```
which should have major.minor in it